src/asciidoc/docs/distribute/app-store.adoc

```diff
@@ -0,0 +1,86 @@
+
+By listing in the Koji App Store, you can take advantage of the traffic in the App Store to get exposure for your app.
+Koji also announces noteworthy apps in social media posts.
+
+This topic shows how to create a Koji App Store listing after you move your app to production.
+
+NOTE: You can have multiple listings for the same app.
+
+== Creating an App Store listing
+
+You need the following for your App Store listing:
+
+* A 500x500-pixel icon to be displayed with your listing in the App Store.
+* A 1200x630-pixel thumbnail to be displayed when your listing is shared in other social media sites.
+* Up to 10 videos and screenshots of your app.
+* The production URL of your app.
+To get this URL, go to the app's associated *Project Details*, click *View Published Koji*, and copy the URL from the browser's address bar.
+
+To create your App Store listing,
+
+include::{includespath}/steps-dev-portal.adoc[tag=go2-dev-portal-templates]
+
+. Click *New App Template*.
+
+. Give the app template a name.
++
+This name is not published in the App Store; it is used to identify your listing.
+
+. Fill in the *App Template Details*.
++
+IMPORTANT: All texts and images in your listing must meet https://withkoji.com/resources/terms[Koji's Terms of Use] requirements.
++
+* *Icon* – The 500x500-pixel icon displayed with your listing.
+
+* *Name* – The title of your listing.
+Note that the title could be truncated depending on the width of the browser page; therefore, place distinguishing words near the beginning.
+
+* *Description* – Features and benefits of your app.
+
+* *Share Image* – The 1200x630-pixel thumbnail that is displayed when your app listing is shared in other social media sites.
+
+* *Videos and Screenshots* – Up to 10 videos and screenshots of your app.
+
+* *Bundled App* – The production URL of your app.
+
+* *Price to Remix* – The price (in US dollars) that the creator/influencer pays to publish their customized version of your app.
+Customization is free; creators/influencers are charged only when they publish their customized version in their Koji profile.
+You keep 85% of this amount; Koji gets 15%.
+Leave blank to allow the creator/influencer to publish their customized version for free.
+
+* *Transaction Fee Percent* – The percentage you wish to collect from each transaction if your app allows In-App Purchases.
+You keep 60% of this percentage; Koji gets 40%.
+Leave blank to collect the default percentage (15%).
++
+For example, if you set this to 10% and the user paid $10 in the app,
++
+** The creator/influencer who added your app to their profile receives $9.00.
+** You receive $0.60.
+** Koji receives $0.40.
+
+* *This template is open source.* – If enabled, other developers can view and fork your app's source code.
+
+* *This template is intended primarily for developers to fork.* – If enabled, your app listing will be placed in the *For Developers* category of the App Store.
```

Comment:
Does it actually go this this category automatically? Or do we need to add something like, when it is approved?

Reply:
Adding the listing to the App Store is not automated. I think "when it is approved" is implied. If it's not approved, it won't appear anywhere in the App Store.

This is one of my guesses because I don't really know how we use this field. I'm just assuming Luis looks at this value.

Reply:
In that case, can you double-check with Luis and/or Sean to verify how this works?

Reply:
Checked with Luis; he doesn't consider that setting when determining which category to put the listing under. However, he said that, if the setting is enabled, the listing gets the *Fork* button instead of a *Use* button.

BEFORE: "If enabled, your app listing will be placed in the *For Developers* category of the App Store."
AFTER: "If enabled, your app listing will include a btn:[Fork] button (instead of a btn:[Use] button) and the message:
+
____
This app is designed for developers to use when creating new apps.
____

Reply:
Great, thanks for checking! I learned something new :)